forch/forchestrator.py

```diff
@@ -37,13 +34,17 @@
 
 LOGGER = logging.getLogger('forch')
 
-_FORCH_CONFIG_DEFAULT = 'forch.yaml'
 _STRUCTURAL_CONFIG_DEFAULT = 'faucet.yaml'
 _BEHAVIORAL_CONFIG_DEFAULT = 'faucet.yaml'
 _SEGMENTS_VLAN_DEFAULT = 'segments-to-vlans.yaml'
 _DEFAULT_PORT = 9019
 _PROMETHEUS_HOST = '127.0.0.1'
 
+
+# TODO: 1) use protobuf after entire forch config is converted
```

Comment:
Each item should have it's own TODO (they are automatically indexed), and should be short enough to be on one line.

Reply:
Done